src/libs/nav.js

```diff
@@ -105,10 +105,8 @@ export const Router = () => {
   ])
 }
 
-export const updateSearch = (query, params) => {
-  const newSearch = qs.stringify({
-    ...query, ...params
-  }, { addQueryPrefix: true })
+export const updateSearch = params => {
+  const newSearch = qs.stringify(params, { addQueryPrefix: true })
```

Comment:
There was no need for this function to take two arguments. If there are two objects, the calling function can merge them.